README.md

```
@@ -0,0 +1,834 @@
+templates:
+  - name: actor/src/pages/index.tsx
+    factoryExpression: "#getPagesForRouting(#application)"
+    pathExpression: "'src/pages/' + #pagePath(#self) + '/index.tsx'"
+    templateName: actor/src/pages/index.tsx.hbs
+    templateContext:
+      - name: page
+        expression: "#self"
+```
+
+**Template Properties:**
+| Property | Description |
```

markdownlint-cli2 (0.20.0)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)